sopel/modules/reddit.py

```diff
@@ -328,3 +378,37 @@ def get_channel_spoiler_free(bot, trigger):
         bot.say('%s is flagged as spoiler-free' % channel)
     else:
         bot.say('%s is flagged as spoilers-allowed' % channel)
+
+
+@rule(r'^(?P<prefix>r|u)/(?P<id>[a-zA-Z0-9-_]+)')
+def reddit_slash_info(bot, trigger):
+    searchtype = trigger.group('prefix')
```

Comment:
This needs to change to make case-insensitive matches work properly:

```suggestion
    searchtype = trigger.group('prefix').lower()
```